src/br/com/caelum/cdc/servlet/AuthorAddServlet.java

```diff
@@ -37,18 +30,21 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
 	@Override
 	protected void doPost(HttpServletRequest request, HttpServletResponse response)
 			throws ServletException, IOException {
-
+		Connection connection = (Connection) request.getAttribute("connection");
+		BindingResult result = (ConcreteBindingResult) request.getAttribute("bindingResult");
```

Comment:
segura esse spring da massa! E realmente não teria outro jeito de buscar esses objetos se não fosse dentro do doPost mesmo. Inclusive que a Servlet é uma instância única... quando for criar atributos, eles não podem específicos de um usuário ou algo assim.